index.html

```diff
@@ -187,6 +187,29 @@ <h2>Top-level Inventory Checksum</h2>
     <section id='versions-directories'>
       <h2>Versions Directories</h2>
 
+      <p>
+	OCFL object content is stored as a sequence of one or more versions. Each object version is stored in a version
+	directory under the object root. The sequence of version numbers is the sequence of positive integers: 1, 2, 3, etc.,
+	and the version directory name is constructed by adding the prefix <code>v</code>.
+      </p>
+      <p>
+	It is RECOMMENDED that implementations use version directory names constructed without zero-padding the version
+	number, ie. <code>v1</code>, <code>v2</code>, <code>v3</code>, etc..
+      </p>
+      <p>
+	For compatibility with existing filesystem conventions, implementations MAY use zero-padded version numbers,
+	with the following restriction: If zero-padded version numbers are used then they MUST start with a zero. For example,
+	in an implementation that uses five digits the version directory names <code>v00001</code> to <code>v09999</code> are
+	allowed, <code>v10000</code> is not allowed.
```

Comment:
To make follow-suit numbering of an inherited filesystem easier. Looking at the last version tells you what the next version number should be - rather than tracking back to the last-version-that-changed-the-number-of-significant-digits.

Reply:
However, we do need to mention follow suit. Version numbers should either be all unpadded or all padded to the same length. Or else we need additional rules so that v1, v001 and v00000000001 don't coexist.

Reply:
...people *will* do that. We have shelfmarks where punctuation and spacing are significant.

Reply:
@ahankinson - I included the `v0...` requirement for padded following the discussion in https://github.com/OCFL/spec/issues/2#issuecomment-388003916

@neilsjefferies - The following part of the PR does sat the the length must be consistent
> All versions MUST use the same naming convention: either a non-padded version number, or a zero-padded version number of consistent length.